polyfill/lib/ecmascript.mjs

```diff
@@ -145,10 +146,16 @@ export function ToIntegerIfIntegral(value) {
   return number;
 }
 
+// This function helps with DRY for error messages, although it's not in the spec
+export function RequireString(value) {
+  if (Type(value) !== 'String') throw new TypeError(`expected a string, not ${StringCtor(value)}`);
```

Comment:
I realize that `StringCtor` is used here so that symbols won't cause this expression to throw a different TypeError than the one we intend to throw. But I don't find it obvious, and it looks redundant, so it may be worth a comment to prevent well-intentioned removal in the future.

Reply:
Good idea. Added.